index.js

```diff
@@ -239,7 +243,12 @@ class K8sVMExecutor extends Executor {
             LOW: this.lowCpu,
             MICRO: this.microCpu
         };
-        const CPU = (cpuConfig in cpuValues) ? cpuValues[cpuConfig] : cpuValues.LOW;
+        let cpu = (cpuConfig in cpuValues) ? cpuValues[cpuConfig] : cpuValues.LOW;
+
+        // allow custom cpu value
+        if (Number.isInteger(cpuConfig)) {
+            cpu = cpuConfig > this.maxCpu ? this.maxCpu : cpuConfig;
```

Comment:
`cpu = Math.min(cpuConfig, this.maxCpu)` is concise. Same for memory as well.

Reply:
good suggestion! addressed